frontends/pytorch/csrc/builder/module_builder.cpp

```diff
@@ -100,9 +99,9 @@ ModuleBuilder::importFunction(torch::jit::StrongFunctionPtr function) {
   auto inserter = createInserter();
   GraphImporter::MlirMappingOptions mappingOptions{
       context,
-      llvm::None, // genericFuncName (default to auto)
-      llvm::None, // funcName (default to auto)
-      typeMapper, inserter,
+      c10::nullopt, // genericFuncName (default to auto)
+      c10::nullopt, // funcName (default to auto)
+      typeMapper,   inserter,
```

Comment:
nit: weird extra space

Reply:
Weird. Removed the trailing comma and clang-format had a better time.